pkg/speccheck/testing/stuff_for_tests.go

```diff
@@ -0,0 +1,20 @@
+package testing
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type FakeStore struct {
+	Namespace string
+	Responses map[string]runtime.Object
+}
+
+func (f FakeStore) Get(gvk schema.GroupVersionKind, namespace, name string) (runtime.Object, bool /*exists*/, error) {
+	if f.Namespace != namespace {
```

Comment:
Currently `Namespace` field is not being used anywhere and namespace is not set for objects in tests. I'm planning to send a follow up PR to set it to have tests that verify that namespace is being passed correctly.